packages/components/src/Stepper/Stepper.component.js

```diff
@@ -37,7 +37,11 @@ export const isErrorInSteps = steps =>
  * @param {array} steps array of steps
  */
 export const isAllSuccessful = steps =>
```

Comment:
Not sure we can consider this as successful anymore, do you mean 
```suggestion
export const isEnded = steps =>
```

Reply:
your request change looks like a breaking change